VirtualReality/Widgets/qMRMLVirtualRealityView.cxx

```diff
@@ -163,6 +164,10 @@ void qMRMLVirtualRealityViewPrivate::createRenderWindow()
   this->Interactor->SetInteractorStyle(this->InteractorStyle);
   this->InteractorStyle->SetInteractor(this->Interactor);
   this->InteractorStyle->SetCurrentRenderer(this->Renderer);
```

Comment:
Given that the `CurrentRenderer` is being set here, IMO, we can remove the `FindPokedRenderer` calls above, no?